service-api/module/Application/src/Services/MockNinoService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Services;
+
+use Application\Services\Contract\NINOServiceInterface;
+
+class MockNinoService implements NINOServiceInterface
+{
+    public function validateNINO(string $nino): string
+    {
+        if (str_ends_with($nino, 'C')) {
+            return 'No Match';
```

Comment:
These should return constant-defined codes (e.g. `NO_MATCH`) rather than text so that we're not tying content to code.